From 9dcf72e144adac9d6401a5794b79e37f8a72a5fa Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 13 Jul 2017 21:17:16 +0200
Subject: [PATCH 4/5] efi_loader: debug info in efi_convert_device_path_to_text

The debug information provided by efi_convert_device_path_to_text
is insufficient. The type and the subtype are needed to find
out why the function did not support a conversion.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_device_path_to_text.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/efi_loader/efi_device_path_to_text.c b/lib/efi_loader/efi_device_path_to_text.c
index a5ea63300b..56de328286 100644
--- a/lib/efi_loader/efi_device_path_to_text.c
+++ b/lib/efi_loader/efi_device_path_to_text.c
@@ -41,6 +41,9 @@ uint16_t *efi_convert_device_path_to_text(
 		return NULL;
 	}
 
+	debug("EFI: type %d, subtype %d\n",
+	      device_path->type, device_path->sub_type);
+
 	switch (device_path->type) {
 	case DEVICE_PATH_TYPE_MESSAGING_DEVICE:
 		switch (device_path->sub_type) {
-- 
2.11.0

